README.md

```diff
@@ -24,6 +24,18 @@ composer require --dev roave/infection-static-analysis-plugin
 vendor/bin/roave-infection-static-analysis-plugin
 ```
 
+### Configuration
+
+The `roave-infection-static-analysis-plugin` binary accepts all of `infection` flags and arguments, and an additional `--psalm-config` argument.
+
+Using `--psalm-config`, you can specify the psalm configuration file to use when analysing generated mutation,
+or the directory containing the psalm configuration file:
+
+```sh
+vendor/bin/roave-infection-static-analysis-plugin --psalm-config config/psalm.xml # configuration file.
+vendor/bin/roave-infection-static-analysis-plugin --psalm-config="config/" # directory containing the configuration file.
```

Comment:
IMO we shouldn't support both formats: let's just support the file format.

Reply:
supporting both doesn't add any complexity in our end, see the if(is_file) in the binary, so i think we can keep it.

Reply:
I'd certainly remove docs for this one though

Reply:
done